docs/settings.rst

```diff
@@ -0,0 +1,14 @@
+SiteSettings
+============
+
+SiteSettings module allows your users to change common shop settings from dashboard like its name or domain.
+Settings object is chosen by pk from ``SITE_SETTINGS_ID`` variable.
+
+
+Context processor
+-----------------
+Thanks to ``saleor.site.context_processors.settings`` you can access SiteSettings in template with ``settings`` variable.
+
+New fields
```

Comment:
I mean this is true for all Django models yet we do not mention anywhere what to do to add fields to other objects. And I don't think we should as this crosses into "how do I use Django" territory.